README.md

```diff
@@ -3,20 +3,43 @@ pyproj
 
 Installation
 ------------
+
 * clone github repo or download source release at http://python.org/pypi/pyproj.
-  * If you clone the github repo, [Cython](http://cython.org/) is a dependency.
-* python setup.py build
-* python setup.py install (with sudo if necessary).
+  * [Cython](http://cython.org/) is required for the installation.
+
+Setup PROJ.4
+~~~~~~~~~~~~
+
+To install PROJ.4 insternally in pyproj, these are the dependencies:
+    * C99
+    * C++11
+    * SQLite 3.7
+    * CMake 3.5
 
-To use proj4 lib (and data files) that are already installed on the system, 
+* git submodule init && git submodule update
+* python install_internal_proj4.py
+
+To use PROJ.4 lib (and data files) that are already installed on the system, 
```

Comment:
Does this mean if you have an old proj4 installation, you can still use it (presumably sacrificing the new API you added that uses features from proj4 v6.0.0?)

Reply:
Unfortunately, no. The proj_api.h is deprecated and the behavior changed within the file. I moved all imports to use the new proj.h. But, wheels and  conda are good solutions for this problem.

Reply:
Well, I take the hard "no" back as I haven't tried it and there might be a way.

Reply:
OK, understood.